go/mysql/collations/colldata/uca_contraction_test.go

```diff
@@ -58,7 +56,7 @@ func findContractedCollations(t testing.TB, unique bool) (result []CollationWith
 			continue
 		}
 
-		rf, err := os.Open(fmt.Sprintf("testdata/mysqldata/%s.json", collation.Name()))
+		rf, err := os.Open(fmt.Sprintf("../testdata/mysqldata/%s.json", collation.Name()))
```

Comment:
was this not working before, or maybe a VTROOT issue?

Reply:
It was not working before anymore but that was not noticed since there tests only run when you manually generate the collation data.